components/button.json

```diff
@@ -43,29 +47,69 @@
     "ghost":{
       "#normal":{
         "background":"@theme-button-secondary-normal",
-        "text": "@theme-text-primary-normal"
+        "text": "@theme-text-primary-normal",
+        "border": "@theme-button-secondary-normal"
```

Comment:
Likewise there's no border on ghost buttons

Reply:
Yes, we will need the border token for high contrast support, without high contrast the border is the same as the background color.